src/ostorlab/agent/mixins/agent_report_vulnerability_mixin.py

```diff
@@ -3,11 +3,65 @@
 Definition of the main method to process an entry from the KB - Knowledge Base,
 and emit a vulnerability message.
 """
-from typing import Optional
+import dataclasses
 import enum
+from typing import Optional, List
 
-from ostorlab.agent.mixins.protocols import emit
 from ostorlab.agent.kb import kb
+from ostorlab.agent.mixins.protocols import emit
+from ostorlab.assets import android_aab as android_aab_asset
+from ostorlab.assets import android_apk as android_apk_asset
+from ostorlab.assets import android_store as android_store_asset
+from ostorlab.assets import asset
+from ostorlab.assets import domain_name as domain_name_asset
+from ostorlab.assets import file as file_asset
+from ostorlab.assets import ios_ipa as ios_ipa_asset
+from ostorlab.assets import ios_store as ios_store_asset
+from ostorlab.assets import ipv4 as ipv4_asset
+from ostorlab.assets import ipv6 as ipv6_asset
+from ostorlab.assets import link as link_asset
+
+MAP_ASSETS_NAMES = {
+    ipv4_asset.IPv4: 'ipv4',
```

Comment:
The string thingy should an attribute of each asset. Centralizing it here is change error prone.